test/unit_test/tutorial_test.cc

```diff
@@ -0,0 +1,195 @@
+#ifndef STATIC_LINK_VW
+#  define BOOST_TEST_DYN_LINK
+#endif
+
+#include <boost/test/unit_test.hpp>
+#include <boost/test/test_tools.hpp>
+
+#include "vw.h"
+#include "test_common.h"
+#include <numeric>
+#include <fmt/format.h>
+
+class Simulator
+{
+  float USER_LIKED_ARTICLE = -1.f;
+  float USER_DISLIKED_ARTICLE = 0.f;
+  std::vector<std::string> users{"Tom", "Anna"};
+  std::vector<std::string> times_of_day{"morning", "afternoon"};
+  std::vector<std::string> actions{"politics", "sports", "music", "food", "finance", "health", "camping"};
+  std::string debug_logfile;
```

Comment:
remove debug_logfile, no need for it right now.

Reply:
Yea wasnt sure if we might want it in the future, just took it out